Submitted By: Thomas Pegg <thomasp at linuxfromscratch dot org>
Date: 2005-10-27
Initial Package Version: 5.8.7
Origin: based on perl-5.8.7-libc-1.patch 
Description: this patch adapts some hard-wired paths to the C library and points it to use lib64 instead of lib.

diff -uNr perl-5.8.0.orig/hints/linux.sh perl-5.8.0/hints/linux.sh
--- perl-5.8.0.orig/hints/linux.sh	2002-06-05 23:46:00.000000000 +1000
+++ perl-5.8.0/hints/linux.sh	2003-02-19 16:32:18.000000000 +1100
@@ -51,9 +51,9 @@
 # We don't use __GLIBC__ and  __GLIBC_MINOR__ because they 
 # are insufficiently precise to distinguish things like
 # libc-2.0.6 and libc-2.0.7.
-if test -L /lib/libc.so.6; then
-    libc=`ls -l /lib/libc.so.6 | awk '{print $NF}'`
-    libc=/lib/$libc
+if test -L /lib64/libc.so.6; then
+    libc=`ls -l /lib64/libc.so.6 | awk '{print $NF}'`
+    libc=/lib64/$libc
 fi
 
 # Configure may fail to find lstat() since it's a static/inline
